### DISCUSSION TITLE
[Mono-Symbolicate] .Net Maui iOS crash

### DISCUSSION THREAD

**Discussion by PetrePopescu** [mono/mono#21550] [Q&A]
Is there any support for symbolicating .**Net Maui** mobile apps?

I am trying to use mono-symbolicate tool to work with a **Maui iOS** app but the **bin** folder does not seem to contain any **msym** folder.

Is there any config/workaround to get the **msym** to show up?
[SymbolTestApp.zip](https://github.com/mono/mono/files/9624863/SymbolTestApp.zip)

**Comment by akoeplinger** [comment]
Could you please open the issue in https://github.com/xamarin/xamarin-macios instead? They should know what the current story around symbolication is.

**Reply by PetrePopescu** [reply]
This issue is strictly related to .Net Maui. Symbolication works fine for Xamarin.

**Reply by akoeplinger** [reply]
The repository is the same, we need to track it there.
